### ISSUE TITLE
Where do you freeze the gradient descent?

### ISSUE THREAD

**Issue by ybsave** [LynnHo/DCGAN-LSGAN-WGAN-WGAN-GP-Tensorflow#9]
Hello, I am confused about how do you freeze the gradient descent to the other model. When training d_step, I suppose the generator should be freezed, as f_logit is based on generator and used in d_loss; similarly, when training g_step, I suppose the discriminator should be freezed, as f_logit depens on discriminator.

However, I do not see where you stop those gradients flowing to the unwanted part, either generator or discriminator. Would you please provide some hints for me? Thank you.

**Comment by LynnHo** [comment]
![image](https://user-images.githubusercontent.com/16457298/47828134-ac5a7680-ddbc-11e8-9e8a-f0cf65427d0c.png)

**Comment by ybsave** [comment]
Ah, I see. Thank you so much. I did not notice the var_list limitation!
